packages/protocol/contracts/bridge/IBridge.sol

```diff
@@ -8,18 +8,17 @@ pragma solidity ^0.8.20;
 
 /// @title IBridge
 /// @notice The bridge used in conjunction with the {ISignalService}.
-/// @dev Ether is held by Bridges on L1 and by the EtherVault on L2,
-/// not by token vaults.
+/// @dev Ether is held by Bridges on L1 and L2s.
```

Comment:
There's a couple of `(auto filled)` and `For ERC20 transfers` comments that I think are outdated.

Reply:
remove them.